mmv1/templates/terraform/examples/dialogflowcx_environment_regional.tf.erb

```diff
@@ -0,0 +1,27 @@
+resource "google_dialogflow_cx_agent" "agent" {
+  display_name = "<%= ctx[:vars]["agent_name"] %>"
+  location = "europe-west2"
```

Comment:
Our sweepers typically target `us-central1` (for cleanup if tests fail and leave resources around). Could we change to that region, or do we need to use `europe-west2` here?

Reply:
`europe-west2` because that's what the existing test used 😅. Changed to `us-central1`.